pluto/build/design.ini

```diff
@@ -9,54 +9,54 @@ url = https://fedoraplanet.org/design
     avatar=http://brejc8.fedorapeople.org/avatar.png
     author=brejc8
 
-; # Origin: /home/fedora/nicubunu/.planet.design
-; [http://nicubunu.blogspot.com/feeds/posts/default?alt=rss]
-;     name=Nicu Buculei
-;     feed=http://nicubunu.blogspot.com/feeds/posts/default?alt=rss
-;     avatar=http://nicubunu.ro/gfx/nicu-gotchi-blond.png
-;     author=nicubunu
+# Origin: /home/fedora/nicubunu/.planet.design
+[http://nicubunu.blogspot.com/feeds/posts/default?alt=rss]
+    name=Nicu Buculei
+    feed=http://nicubunu.blogspot.com/feeds/posts/default?alt=rss
```

Comment:
There are a lot of non-HTTPS links in these entries. We should think about whether we should change those to HTTPS where possible.